Emergency disable sccache on OS X.

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#30319 Emergency disable sccache on OS X.**

Our current binaries do not appear to be compatible with
CircleCI's new OS X environment.

See https://github.com/pytorch/pytorch/issues/30321

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

ezyang committed:
Emergency disable sccache on OS X.

Our current binaries do not appear to be compatible with
CircleCI's new OS X environment.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

albanD reviewed (approved):
nit: add a comment in the code stating why this was disabled ?

ezyang committed:
Update on "Emergency disable sccache on OS X."


Our current binaries do not appear to be compatible with
CircleCI's new OS X environment.

See https://github.com/pytorch/pytorch/issues/30321

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

ezyang commented:
Didn't work. Trying a different strat.

ezyang committed:
Update on "Emergency disable sccache on OS X."


Our current binaries do not appear to be compatible with
CircleCI's new OS X environment.

See https://github.com/pytorch/pytorch/issues/30321

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

ezyang commented:
The build is so slow, this may be a case of cure is worse than disease.

ezyang commented:
I'm not planning on landing this diff, but feel free to land it if you think it's appropriate

albanD commented:
Won't be needed with the new sccache that was uploaded on s3 that statically link openssl.
